python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/logging.j2

```diff
@@ -6,6 +6,9 @@
 {# eos - logging #}
 {% if logging is arista.avd.defined %}
 !
+{%     if logging.repeat_messages is arista.avd.defined(false) %}
+no logging repeat-messages
```

Comment:
shall we add `logging repeat-messages` here when its `true`?
@gmuloc

Reply:
yes let's do it